startly_style.css

```diff
@@ -0,0 +1,218 @@
+@import url('http://fonts.googleapis.com/css?family=Economica');
+@import url('http://fonts.googleapis.com/css?family=Monoton');
+@import url('http://fonts.googleapis.com/css?family=Tangerine');
+
+
+* {
+  font-family: Economica;
+  background-color: seashell;
+}
+
+.container {
```

Comment:
I was considering whether or not to put the whole page in a container - was it helpful to make the entire thing a flexbox? Did you use the functionality?